src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -788,6 +792,38 @@ public static double IEEERemainder(double x, double y)
             }
         }
 
+        public static int ILogB(double x)
+        {
+            // Implementation based on https://github.com/ifduyue/musl/blob/cfdfd5ea3ce14c6abf7fb22a531f3d99518b5a1b/src/math/ilogb.c
```

Comment:
Can we switch this to the official MUSL source: https://git.musl-libc.org/cgit/musl/tree/src/math/ilogb.c ?

At first glance, they appear to be identical and it would be better, IMO, to ensure we use the main source rather than a fork.

Reply:
Sure, makes sense. I've updated the citations, please let me know if there are any other changes I can make.